docs/docs/guides/build-workflows.md

```diff
@@ -86,7 +86,7 @@ example.CHAIN:
     - B
```

Comment:
My suggestion here:

You can use the `periodic > interval` key with a number argument (unity is the second):

```yaml
example.CHAIN:
  tasks:
    - A
    - B
    - C
  periodic:
    interval: 60
```

You can also use the `periodic > crontab` key with a string argument:

```yaml
example.CHAIN_CRONTAB:
  tasks:
    - A
    - B
    - C
  periodic:
    crontab: "0 */3 * * *"
```

In this case the format is the following (the [official documentation](https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#crontab-schedules) of the `crontab` function gives some examples of each attribute):

```yaml
periodic:
  crontab: "minute hour day_of_week day_of_month month_of_year"
```

Reply:
Done